src/markdown-pages/docs/managers/mq-manager.md

```diff
@@ -0,0 +1,164 @@
+---
+path: "/docs/managers/mq-manager"
+title: "MQ Manager"
+---
+
+**Alpha**
+
+## Overview
+This Manager provides the ability to connect to an existing Queue Manager, enabling applications to read and write one or more messages to and from a queue. <br><br> 
```

Comment:
Need to explain it's for a IBM MQ queue manager

Reply:
Have added in extra content